libs/core/src/lib/form/form-group/form-group.component.ts

```diff
@@ -20,28 +20,31 @@ import { CssClassBuilder, applyCssClass } from '../../utils/public_api';
     styleUrls: ['./form-group.component.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class FormGroupComponent implements CssClassBuilder, AfterViewInit {
-    class: string;
-    /** @hidden */
+export class FormGroupComponent implements CssClassBuilder, OnChanges, OnInit {
     @HostBinding('class.fd-form-group')
     fdFormGroupClass: boolean = true;
 
-    private _isInline: boolean = false;
     /** Determines if form items should be displayed inline or not
      * Default value is set to false;
      */
-    @Input() set isInline(inline: boolean) {
-        this._isInline = inline;
-        this.buildComponentCssClass();
-    }
+    @Input()
+    isInline: boolean;
+
```

Comment:
These changes are also introduced in https://github.com/SAP/fundamental-ngx/pull/2228